body.tex

```diff
@@ -692,7 +700,98 @@ \subsection{Reporting Cycle}
+This sandbox workshop at the end of May / beginning of June gives all the departments, including the Director's Office, a chance to review, and probe, the ground-up planning in each team.
+(In practice, only the four technical departments' management teams are likely to sit through the entire workshop, although it is organized inclusively so that anyone could choose to review all the scrubbed plans.)
+
+In most cases a team's proposed changes in effort distribution, staffing, and purchasing will either involve only small deviations from budget guidance, or stay within the ``box'' given to them.
+Teams may also coordinate their proposed changes so as to keep the department's overall spending plan within its budget.
+(This coordination may be facilitated by the department's management team, e.g.\ following the pitch sessions.)
+
+Following the Sandbox Workshop, the Director's Office adjusts and finalizes the overall spending plan by the end of June, in time for the submission of the NOIRLab POP, and SLAC's DOE budget briefing.
+This June period of post-sandbox spending plan development and cost calculation involves the Directors Office staff making updates to the planning tools (Appendix~\ref{sec:planning-tools}), with review by teh departmental management teams.
+
+The changes to the spending plan, once captured in the Rubin planning tools, are implemented in NOIRLab's and SLAC's lab-wide budget models in July.
+Statements of work for the next fiscal year's contracts and MPOs are straightforwardly derived from the scrubbed plans, for their July deadlines.
+
+In September, the teams then plan work (for the first half of the new fiscal year) against their minor milestones, incorporating any community input they may have collected at the annual project and community workshop (PCW) in August.
+
+\begin{figure}[!ht]
+\includegraphics[width=0.95\linewidth]{figures/annual-budget-cycle.png}
+\caption{Annual budget planning cycle.}
+\label{fig:annual-cycle}
+\end{figure}
+
+
+\subsection{Risk-aware budgeting and Requests Beyond Target}
+
+The ground-up spending plan developed during the annual cycle is to cover operations work defined by the milestone activities, as well as any regular work captured in bucket epics.
+This work includes mitigation of risks held by the teams, as described in the response plans in the risk register.
+Any risks that are realized during the year will require action that is not budgeted for.
+Additional spending to address realized risks is enabled by Rubin's request beyond target (RBT) process, whereby reserve funds are released for use by the departments in response to a specific mid-year request made in a custom Jira ticket.
```

Comment:
I think RBT process should be captured in the CCB process - that might be a specific JIRA ticket but it would be one related to the CCB process

Reply:
Yes - let's remember to point this out in the text when we do the CCB update.